x-pack/plugins/aiops/public/hooks/use_eui_theme.ts

```diff
@@ -9,15 +9,22 @@ import { useMemo } from 'react';
 
 import { euiLightVars as euiThemeLight, euiDarkVars as euiThemeDark } from '@kbn/ui-theme';
 
+import { of } from 'rxjs';
+import useObservable from 'react-use/lib/useObservable';
 import { useAiopsAppContext } from './use_aiops_app_context';
 
 export type EuiThemeType = typeof euiThemeLight | typeof euiThemeDark;
 
+const themeDefault = { darkMode: false };
+
 export function useEuiTheme() {
-  const { uiSettings } = useAiopsAppContext();
+  const { theme } = useAiopsAppContext();
```

Comment:
This PR adds three very similar functions to plugins own by us, it would be nice to consolidate them into a common function and add it to a package.
Can be done in a follow up bit of refactoring.